lib/hpcbase.pm

```diff
@@ -115,6 +115,8 @@ sed -i "/^NodeName.*/c\\NodeName=$cluster_ctl_nodes,$cluster_compute_nodes Socke
 sed -i "/^PartitionName.*/c\\PartitionName=normal Nodes=$cluster_ctl_nodes,$cluster_compute_nodes Default=YES MaxTime=24:00:00 State=UP" /etc/slurm/slurm.conf
```

Comment:
some time ago I introduced this function :
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/utils.pm#L1547 , might worth looking for this case . ( just a side note not related to this PR)